### PR TITLE
React Native: Mobile application development tutorial

### PR DESCRIPTION
**Pull request by masum-ulu**
Resolves  https://github.com/volosoft/volo/issues/14262

This PR includes mobile application development tutorial with `react native`

---

@EngincanV Can you please check the structure of folder

### CODE REVIEW

**Review by EngincanV — COMMENTED** [review]
Can you delete the `en/tutorials/mobile/image` folder and move all the images in this folder to the main `images` folder (https://github.com/abpio/abp-commercial-docs/tree/dev/en/images)?

File: en/tutorials/mobile/react-native.md [hunk 1]
```diff
@@ -0,0 +1,1787 @@
...
+## About This Tutorial
+
+- This tutorial assumes that you have completed the [Web Application Development tutorial](../part-1.md) and built an ABP based application named `Acme.BookStore`. In this tutorial, we will only focus on the UI side of the `Acme.BookStore` application and will implement the CRUD operations.
+- Before start please be sure the [React Native Development Environment](https://docs.abp.io/en/commercial/latest/getting-started-react-native) is ready on your machine.
+
+## Download the Source Code
+
+You can use the following link to download the source code of the application described in this article:
+
+- [Acme.BookStore]()
```
Comment (EngincanV) [inline en/tutorials/mobile/react-native.md]: I just wanted to add a comment here to remind you to update the link before merging the PR.

**Review by EngincanV — APPROVED** [review]
The documentation is really good and shows all related code blocks to build up the application. But I think that we need to describe each section plainly, I mean each section needs to have short text that explains what the added code does, in my opinion.

**Comment by masum-ulu** [comment]
> The documentation is really good and shows all related code blocks to build up the application. But I think that we need to describe each section plainly, I mean each section needs to have short text that explains what the added code does, in my opinion.

I've update the tutorial, I tried to add descriptions. Can you please review again when you can @EngincanV
I'll update demo project link after merge https://github.com/volosoft/abp-commercial-samples/pull/54

**Review by mahmut-gundogdu — APPROVED** [review]

**Review by braim23 — APPROVED** [review]

**Pull request merged**